src/main/java/at/laborg/briss/BrissGUI.java

```diff
@@ -84,7 +86,7 @@
  * @author gerhard, hybridtupel
  */
 
-public class BrissGUI extends JFrame implements PropertyChangeListener, ComponentListener {
+public class BrissGUI extends JFrame implements PropertyChangeListener, ComponentListener, MouseListener {
```

Comment:
That would be nice, however as it already extends JFrame I fear that is not possible. At least the last time I checked, Java did not have multiple inheritance.

Reply:
Duh! me 🤦

Reply:
What about an anonymous class? Like this:

```java 
addMouseListener(new MouseAdapter() {
    @Override
    public void mousePressed(MouseEvent e) {
        deselectAllRects();
    }
});
```

Reply:
That's a good idea, have implemented it now that way

Reply:
Great! I'm resolving this now. Thanks!

Reply:
I think now you need to re-review it again and approve it 😄 
It's still blocked with "Changes requested"

Reply:
Ouch! I've never used this workflow myself before. 🤣

Reply:
No worries, I think after the first time you quickly get used to it.
When requesting changes it will block the merge making it necessary to re-review. When just commenting or approving it will not block the merge.